code/modules/holiday/holidays.dm

```diff
@@ -78,7 +82,22 @@
 	begin_month = FEBRUARY
 
 /datum/holiday/birthday/greet()
-	return "Say 'Happy Birthday' to Space Station 13, first publicly playable on February 16th, 2003!"
+	var/game_age = text2num(time2text(world.timeofday, "YY")) - 3
```

Comment:
first playable version was in 2003, game age is only 0 to 99, so you take the three years after Y2K but before spessmen away from the year to get an age #lifehacks